docs/scos/dev/feature-integration-guides/202200.0/multiple-abstract-products-promotions-and-discounts-feature-integration.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Multiple Abstract Products as Promotional Products & Discounts feature integration
```

Comment:
the title and file name are different. Which one is correct?
Both must be identic

Reply:
The title is correct